src/contract_clients/starknet_validity.rs

```diff
@@ -17,7 +16,7 @@ use crate::contract_clients::core_contract::{
 use crate::utils::convert_felt_to_u256;
 
 pub struct StarknetValidityContract {
```

Comment:
I don't have much knowledge about the bootstraper but why isn't it imported from Zaun?

Reply:
@apoorvsadana do you have context of this?

Reply:
we use it

```
pub struct StarknetValidityContract {
    pub core_contract_client: StarknetValidityContractClient,
}
```

`StarknetValidityContractClient` is from zaun

Reply:
we created a wrapper for our use case here

Reply:
ah yup, my bad